docs/user-guide/admin-cli.md

```diff
@@ -433,20 +512,32 @@ optional arguments:
 
 #### Remove
 Remove access to a datastore for a vmgroup.
+An access privilege cannot be removed if the datastore in this privilege is set as "default_datastore" for the vmgroup.
```

Comment:
simpler is better :-)
Removing of access privilege to "default_datastore" is not suported